tests/logs/community_push/test_community_push/test_sharing_accounts_marketplace_format.txt

```diff
@@ -0,0 +1,48 @@
+[    INFO] Loading items from koji:https://fakekoji.com?vmi_build=ami_build
+[    INFO] Retrieving the mappings for ami_pushitem from https://starmap-example.com using the community workflow.
+[    INFO] starmap query returned for ami_pushitem : {"name": "test-build", "version": "7.0", "query_response": {"name": "test-product", "workflow": "community", "clouds": {"aws_storage": [{"id": null, "meta": {"billing-code-config": {"sample-hourly": {"name": "Hourly2", "codes": ["bp-6fa54006"], "image_name": "sample_product", "image_types": ["hourly"]}, "sample-access": {"name": "Access2", "codes": ["bp-63a5400a"], "image_name": "sample_product", "image_types": ["access"]}}, "sharing_accounts": ["account1", "account2", "account3"]}, "architecture": "x86_64", "destination": "fake-destination-access", "overwrite": false, "restrict_version": false, "restrict_major": null, "restrict_minor": null, "provider": "awstest", "tags": null}]}}} 
+[ WARNING] No accounts definition in StArMap, leaving the defaults from credentials.
+[    INFO] Loading sharing_accounts from StArMap.
+[   DEBUG] Loaded the following accounts to the default region: ['account1', 'account2', 'account3']
```

Comment:
This is the message for the standard format, which is the same from `marketplace` workflow:

```
meta:
  sharing_accounts:
    - account1
    - account2
    - account3
```